### ISSUE TITLE
Unable to open and see source files in VSCode when clicking red box stack trace

### ISSUE THREAD

**Issue by ericvera** [Microsoft/vscode-react-native#384]
## Actual Behavior
1. Click on one of the stack frames on the red screen in react native
2. A file opens on the currently open VS Code window (good)
3. The opened tab title shows `<filename>:<line number>` (e.g. `SomeComponent.js:12`), but does not have any content

## Expected Behavior
1. Click on one of the stack frames on the red screen in react native
2. A file opens on the currently open VS Code window
3. The opened tab has the content of the file and the cursor is at the line of the issue

## Software versions
- React-Native VS Code extension version: 0.2.5
- VSCode version: 1.8.1
- OS platform and version: OSX
- NodeJS version: v7.4.0
- React Native version: 0.38.0

## Outputs (Don't think is relevant, but including jus tin case)
```
[vscode-react-native] Debugger ready. Enable remote debugging in app.
node --debug-brk=11092 --nolazy launchReactNative.js 9090 
Debugger listening on 127.0.0.1:11092
[vscode-react-native] Downloaded debuggerWorker.js (Logic to run the React Native app) from the Packager.
[vscode-react-native] Starting debugger app worker.
[vscode-react-native] Established a connection with the Proxy (Packager) to the React Native application
[vscode-react-native] Debugging session started successfully.
```

**Comment by vladimir-kotikov** [comment]
@matrosov-nikita, could you please take a look?

**Comment by matrosov-nikita** [comment]
@vladimir-kotikov, I started to look at this issue.

**Comment by matrosov-nikita** [comment]
@ericvera, it looks like you’re running into known [bug](https://github.com/facebook/react-native/commit/f49093f39710173620fead6230d62cc670570210) in react-native, which will be fixed in 0.41. Also to mitigate the problem when packager has been started from VSCode, I made a PR: https://github.com/Microsoft/vscode-react-native/pull/388.

**Comment by vladimir-kotikov** [comment]
@MSLaguana, can we close this due to #388 has been merged?
